package.json

```
@@ -1358,6 +1354,10 @@
             "description": "%configuration.dotnet.symbolSearch.searchReferenceAssemblies%",
             "order": 80
           },
+          "dotnet.unitTests.runSettingsPath": {
+            "type": "string",
+            "description": "Path to the .runsettings file which should be used when running unit tests."
```

Would be nice if the description retained the O# setting name to make it searchable for some period of adjustment time.
```suggestion
            "description": "Path to the .runsettings file which should be used when running unit tests. (Previously `omnisharp.testRunSettings`)"
```

---

wfm